src/Compilers/Core/Portable/Symbols/IFieldSymbol.cs

```diff
@@ -39,6 +39,13 @@ public interface IFieldSymbol : ISymbol
         /// </summary>
         bool IsVolatile { get; }
 
+        /// <summary>
+        /// Returns true if this field was declared as "fixed".
+        /// Note that for a fixed-size buffer declaration, this.Type will be a pointer type, of which
+        /// the pointed-to type will be the declared element type of the fixed-size buffer.
+        /// </summary>
+        bool IsFixed { get; }
```

Comment:
Language calls these "Fixed size buffers": https://github.com/dotnet/csharplang/blob/master/spec/unsafe-code.md#fixed-size-buffers

Perhaps the name shoudl be IsFixedSizeBuffer to reflect that?

Reply:
I went with `IsFixed` because the existing implementation of the C# version of `FieldSymbol` already uses `IsFixed` to indicate fixed size buffers.

I could rename the interface property to `IsFixedSizeBuffer`, but that would lead to either a redundant property on C#'s `FieldSymbol` or a messy refactoring to rename the existing property on C#'s `FieldSymbol`. Of those, I think I like the redundant property best if we go this route. Thoughts?

Reply:
You can just use explicit interfface implementation to implement the C# version (which forwards to the existing implicit implementation).

But hold off until the compiler people weigh in. They may prefer the name as is.  @gafter and @jcouv are good resources here for helping determine what would be the most appropriate public name :)

Reply:
The compiler already chose to use `IsFixed` for the name here.  I don't think having this PR change that decision is right. I'd stick with the current language.

Reply:
> The compiler already chose to use IsFixed for the name here. I don't think having this PR change that decision is right. I'd stick with the current language.

I'm not sure that's relevant :)   The API is currently internal.  But this PR changes it to be public.  As such, it should be based on what's best given the langauge construct it represents.  Now, we may decide that's 'IsFixed'.  But it should be an actual decision, and not just choosing it based on what the compiler happens to have chose so far.  :)

As another example where this would certainly come up.  If we wanted to expose TypeWithAnnotations, we likely wouldn't call it that name.  Often times internal names are just picked for expediency, or because that specific dev thought it made sense, but without thought given to if that name would be best as a public eye.  When promoting internal to public we need to act without that biasing our decisions.

Reply:
> But it should be an actual decision, and not just choosing it based on what the compiler happens to have chose so far. :)

I looked at the change and it met my expectations. Another compiler sign off is required hereand if they disagree that's fine. I think this name is just fine though.

Reply:
@jaredpar I actually disagree.

The language has a specific notion of `fixed` meaning "cannot move in memory and does not require a `fixed` statement".

I think we misnamed this API and it should be called `IsFixedSizeBuffer`.